src/cirq_iqm/iqm_client.py

```diff
@@ -108,11 +108,11 @@ def from_dict(inp: dict[str, Union[str, dict]]) -> RunResult:
         return RunResult(status=RunStatus(input_copy.pop("status")), **input_copy)
 
 
-class IQMBackendClient:
-    """Provides access to a remote IQM quantum device.
+class IQMClient:
+    """Provides access to an IQM quantum computer.
```

Comment:
```suggestion
    """Provides access to quantum computers through the IQM computation control service.
```

Reply:
I see your point here but it's not so easy to keep it consistent across all the modules... It seems that "server/server interface" and "quantum computer" is the least confusing way to explain what we have here. Additional concepts such as a service, however real, only seem to complicate the documentation.

Also, we should be proud of our main product at IQM, the quantum computers :)